diff -up python-ets-4.4.4/setup_data.py.1~ python-ets-4.4.4/setup_data.py
--- python-ets-4.4.4/setup_data.py.1~	2024-07-07 19:45:05.508827725 +0200
+++ python-ets-4.4.4/setup_data.py	2024-07-07 19:46:59.662557369 +0200
@@ -19,5 +19,5 @@ requirements = [
 INFO = {
     'name': 'ets',
     'version': '4.4.4',
-    'install_requires': ['%s >= %s.dev' % nv for nv in requirements],
+    'install_requires': ['%s >= %s' % nv for nv in requirements],
 }
